class/bike/Bike.js

```diff
@@ -79,20 +78,41 @@ export class Bike extends EventEmitter {
         }
     }
 
-    update(left, right, up, down) {
+    update(progress, delta) {
+        if (this.slow) {
+            progress = (progress + this.slowParity) / 2;
+        }
+        this.backWheel.update(progress);
+        this.frontWheel.update(progress);
+        this.head.update(progress);
+        if (this.upPressed) {
+            this.distance += this.backWheel.rotationSpeed * delta / 100;
+        }
+    }
+
+    setSlow(val) {
+        if ((!this.slow && val) || !val) {
+            this.slowParity = 0;
+        }
+        this.slow = val;
+    }
+
+    fixedUpdate(left, right, up, down) {
         if (this.backWheel.driving && this.frontWheel.driving) {
-            this.slow = false;
+            this.setSlow(false);
         }
-        // iterate twice if not slowed or dead
-        for (let j = 0, i = this.slow || this.dead ? 1 : 2; j < i; j++) {
-            if (!this.dead) {
-                this.updateControls(left, right, up, down);
-            }
+        if (!this.dead) {
+            this.updateControls(left, right, up, down);
+        }
+        if (this.slow) {
+            this.slowParity = (this.slowParity + 1) % 2;
+        }
+        if (!this.slow || this.slowParity === 0) {
```

Comment:
`!this.dead` should be in this statement to match the old for loop

Reply:
I put it back in, but not quite in the same way. I made it so that when you die, `this.slow` is set to true, and now if you are dead, touching both wheels doesn't disable slow motion.
I also hard-coded slow motion into DeadBody and Shard